lisa/tools/make.py

```diff
@@ -6,7 +6,8 @@
 
 from lisa.executable import Tool
 from lisa.operating_system import Posix
-from lisa.tools import Gcc, Lscpu
+from lisa.tools import Gcc
+from lisa.tools.lscpu import Lscpu
```

Comment:
What's the reason to split the import to two?

Reply:
After import Make in kdump.py, here have a circle import. Changing like this, no circle import error.